manifests/m/Microsoft/WebDeploy/3.6/Microsoft.WebDeploy.yaml

```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: Microsoft.WebDeploy
+PackageVersion: 3.6

```

Comment:
```suggestion
PackageVersion: 10.0.1973
```

Reply:
I am sorry, the version in the MSI package is incorrect - the real one is 3.6

Reply:
Yet that is the one you HAVE to use otherwise winget will not work as expected.
All the fields i requested changes on have to be what i changed it to.